app/use_cases/enrich_content.rb

```diff
@@ -22,17 +22,48 @@ def execute
 
   private
 
-  def to_number(value)
-    return if value.blank?
+  def to_number(value, qids)
+    return nil if value.blank?
 
-    (value.to_f % 1).positive? ? value.to_f : value.to_i
+    my_value = determine_numeric_value(value, qids)
+    # if we are a string that can't be converted to a number,
+    # return nil so we are registered as a missing value
+    return nil if my_value.is_a?(String) && (my_value =~ /\A-?\.?[0-9]/).blank?
+
+    (my_value.to_f % 1).positive? ? my_value.to_f : my_value.to_i
```

Comment:
No that doesn't work here because `my_value` can be a string. So you first need to convert it from string to number, with either to_f or to_i.
And anything that you convert from string to number with .to_f fails the .integer? test:
![ConEmu64_2020-02-20_16-58-38](https://user-images.githubusercontent.com/607965/74953679-4014e380-5402-11ea-816f-4ec80c73cc11.png)